src/main/java/codegym/tequila/fisioapp/model/Patient.java

```diff
@@ -0,0 +1,35 @@
+package codegym.tequila.fisioapp.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "patient")
+public class Patient {
+
+    @Id
+    private String id;
+
+    private String name;
+
+    @OneToOne(mappedBy = "patient")
+    private MedicalRecord medicalRecord;
```

Comment:
It's not necessary to add the relation in both classes, the Entity Patient is not dependent on MedicalRecord and it could omit this field. 

I recommend omit it for now and we'll add it if we have a use case in the future.